src/app/containers/ATIAnalytics/atiUrl/index.js

```diff
@@ -228,16 +227,16 @@ export const buildATIEventTrackUrl = ({
     {
       key: 'ati',
```

Comment:
Based on http://bbc.github.io/echo-docs/documentation/reporting-ati/#user-action-events (the code extract at the bottom) put together with soome of the event tracking documentation on confluence, this will always come through in ATI as a "background" event (ie the page has loaded). In order to get it to be an "interaction" event, we need to send it in the `atc` parameter.

Reply:
+1 for creating a ticket to handle this

Reply:
I've created #4819